design/client.md

```diff
@@ -0,0 +1,12 @@
+# Client
+
+Since containerd uses gRPC, it should be easy to implement 3rd-party containerd client in almost any language.
+However, 3rd-party client would need to reimplement pull&push interactions with remote registries, as these are not implemented on the daemon side.
```

Comment:
are push&pull interactions the only thing a 3rd-party client would need to reimplement?

Reply:
No, there is a huge list. We should probably document that list here.